src/umpire/CMakeLists.txt

```
@@ -49,7 +49,7 @@ if (UMPIRE_ENABLE_CUDA AND UMPIRE_ENABLE_DEVICE_ALLOCATOR)
                               PROPERTIES
                               LANGUAGE CUDA)
 
-  set(umpire_device_depends camp cuda)
+  set(umpire_device_depends camp cuda umpire_util)
```

Why umpire_util?  The helper seems to be calling `umpire::fmt::format`, not util.  Shouldn't this be:
```suggestion
  set(umpire_device_depends camp cuda umpire_tpl_fmt)
```

---

I just tested that and it works with that as well. @davidbeckingsale should I change it to `umpire_tpl_fmt` instead?

---

The error classes are provided under umpire_util so I think that is what should be used. The fmt dependency will get brought in transitively.